jina/serve/networking.py

```diff
@@ -52,6 +53,32 @@ class _NetworkingMetrics:
     send_requests_bytes_metrics: Optional['Summary']
 
 
+@dataclass
+class _NetworkingHistograms:
```

Comment:
```suggestion
class _NetworkingMetrics:
```
I am not a big fan to tied to class name to a type of object( i,e, Histograms here)) because the underlying implementation can change (e.g, using a nother type of Hisotgram or Summary in the future) but the goal of the class is the same.

Reply:
Hmm my bad

Reply:
lets keep it like this then